source/TitleState.hx

```diff
@@ -590,16 +590,16 @@ class TitleState extends MusicBeatState
 			{
 				case 1:
 					#if PSYCH_WATERMARKS
-					createCoolText(['Psych Engine by'], 15);
+					createCoolText(['Psych Engine But Better by'], 15);
 					#else
 					createCoolText(['ninjamuffin99', 'phantomArcade', 'kawaisprite', 'evilsk8er']);
 					#end
 				// credTextShit.visible = true;
 				case 3:
 					#if PSYCH_WATERMARKS
-					addMoreText('Shadow Mario', 15);
-					addMoreText('RiverOaken', 15);
-					addMoreText('bb-panzu', 15);
+					addMoreText('Luke', 15);
+					addMoreText('Jamie', 15);
+					addMoreText('Rosie', 15);
```

Comment:
WHO ARE YOU PEOPLE????
![wh](https://user-images.githubusercontent.com/58280776/148662325-3abb7143-07e7-45f7-88d4-86b8b1747d1a.gif)